subsys/net/openthread/Kconfig.defconfig

```diff
@@ -34,12 +34,10 @@ if SOC_NRF54H20_CPUAPP
 
 #To be removed
 config MBOX
-	bool
 	default y
 
 #To be removed
 config MBOX_NRFX_IDS
-	bool
 	default y
```

Comment:
@frkv  are `MBOX` and `MBOX_NRFX_IDS` still needed? I can see that they have been added by Aegir team

Reply:
already removed in other commits